docs/user-guide/image_alignment.ipynb

```diff
@@ -0,0 +1,143 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Image Alignment\n",
+    "\n",
+    "In case of displacement of the detector component, i.e. scintilator,\n",
+    "the open beam (background) image can not be used\n",
+    "for normalization of the sample (signal) image without alignment.\n",
+    "\n",
+    "Here is a simple image position shifting solution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "\n",
+    "ob_image = Path('background.tiff')\n",
+    "misaligned_image = Path('misaligned_sample.tiff')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load tiff image\n",
+    "from PIL import Image\n",
+    "\n",
+    "ob = Image.open(ob_image)\n",
+    "misaligned = Image.open(misaligned_image)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "ob_normalized = (ob / np.max(ob) * 255).astype(np.uint8)\n",
+    "misaligned_normalized = (misaligned / np.max(misaligned) * 255).astype(np.uint8)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def shift_pixels(image: np.ndarray, shift_x: int = 0, shift_y: int = 0) -> np.ndarray:\n",
+    "    \"\"\"Shifts the pixels of an image by ``shift_x`` and ``shift_y``.\n",
+    "\n",
+    "    Image size will remain the same, shifted pixels will\n",
+    "    be cropped and empty pixels will be filled with zeros.\n",
+    "    \"\"\"\n",
+    "    rows, cols = image.shape\n",
+    "    empty_image = np.zeros(image.shape, dtype=np.uint8)\n",
+    "    cropping_point = (\n",
+    "        (max(0, -shift_y), min(rows, rows - shift_y)),\n",
+    "        (max(0, -shift_x), min(cols, cols - shift_x)),\n",
+    "    )\n",
+    "    placing_point = (\n",
+    "        (max(0, shift_y), min(rows, rows + shift_y)),\n",
+    "        (max(0, shift_x), min(cols, cols + shift_x)),\n",
+    "    )\n",
+    "    empty_image[\n",
+    "        placing_point[1][0] : placing_point[1][1],\n",
+    "        placing_point[0][0] : placing_point[0][1],\n",
+    "    ] = image[\n",
+    "        cropping_point[1][0] : cropping_point[1][1],\n",
+    "        cropping_point[0][0] : cropping_point[0][1],\n",
+    "    ]\n",
+    "    return empty_image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "fig, ax = plt.subplots(2, 2, figsize=(15, 10))\n",
+    "im = ax[0, 0].imshow(ob_normalized)\n",
+    "ax[0, 1].imshow(misaligned_normalized, cmap=im.cmap)\n",
```

Comment:
Loading the data as a `DataArray` would then just use Plopp for plotting. This would also mean one could make use of other tools such as the instrument view.

But again, not sure much would be left in the notebook if shifting the positions is a simple vector addition.

Reply:
I think it doesn't matter even if it's short.
I'll rewrite it again though